src/server/html.ts

```diff
@@ -14,7 +15,9 @@ const html = ({ body, helmet }: { body: string, helmet: HelmetData }): string =>
       ${helmet.title.toString()}
       ${helmet.meta.toString()}
       ${helmet.link.toString()}
+      <link rel="shortcut icon" type="image/png" href="${icon}"/>
       <link rel="stylesheet" href="${base}public/bundle.css" />
+      <meta name="viewport" content="width=device-width, initial-scale=1">
```

Comment:
Nitpicking, but grouping <link> and <meta> tags together makes it better organized (with the exception of the charset <meta> tag which should come before the <title> if we need to have it.

So `${helmet.meta.toString()}` could be moved below the latest <link> and above the viewport <meta>

Reply:
Fair enough, I like things to be tidy as well